### PR TITLE
Molden exporting enabled

### PR DESCRIPTION
**Pull request by Madu86**
In this PR, as requested in issue #251, I have enabled capability to export data from QUICK into molden format. Users can put `EXPORT=MOLDEN` in input file and this would result in input_name.molden file in the working directory. Ive followed this article: https://www.theochem.ru.nl/molden/molden_format.html for molden format, however, molden and other visualization software Ive tried dont read SCF/geometry convergence data. Interestingly, other QM codes (eg. MOLCAS, PSI4) do not write these sections into molden files. Not sure why.. Below are some orbital images generated using QUICK's molden files and JMOL. 

Screen Shot 2022-07-22 at 11 00 12 AM
Screen Shot 2022-07-22 at 11 00 33 AM
Screen Shot 2022-07-22 at 11 02 55 AM
Screen Shot 2022-07-22 at 11 03 09 AM

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,3 +15,4 @@ debug.*
 *~
 *.bak
 make.in
+quick.rc
```
Comment (agoetz) [inline .gitignore:18]: I think we need quick.rc, the one in `quick-cmake/`. Shouldn't ignore it.

File: src/modules/quick_method_module.f90 [hunk 2]
```diff
@@ -755,6 +756,13 @@ subroutine read_quick_method(self,keywd,ierr)
             if (index(keyWD,'COARSEINT').ne.0) self%coarse_cutoff=.true.
             if (index(keyWD,'TIGHTINT').ne.0) self%tight_cutoff=.true.
 
+            ! read and enable exporting
+            if (index(KeyWD,'EXPORT=') .ne. 0) then
+                tempstring=' '
+                call read(keyWD, 'EXPORT', tempstring)
+                if(index(tempstring,'MOLDEN') /= 0) write_molden=.true.
```
Comment (agoetz) [inline src/modules/quick_method_module.f90:763]: Should we throw an error if there is a typo/unknown export format?

**Review by agoetz — CHANGES_REQUESTED** [review]
Looks all good with one exception. When doing a geometry optimization, we should export the optimized geometry along with the MOs of the optimized geometry. Currently we are exporting the initial geometry.

File: .gitignore (hunk 1 above)
Comment (Madu86) [reply .gitignore]: Fixed.

File: src/modules/quick_method_module.f90 (hunk 2 above)
Comment (Madu86) [reply src/modules/quick_method_module.f90]: Fixed.

**Comment by Madu86** [comment]
> Looks all good with one exception. When doing a geometry optimization, we should export the optimized geometry along with the MOs of the optimized geometry. Currently we are exporting the initial geometry.

Please see my original note. I have implemented a subroutine to write geometries along an optimization: https://github.com/Madu86/QUICK/blob/molden/src/modules/quick_molden_module.f90#L229-L262 but for some reason molden doesnt read them. Do you have an example molden file I can look at? The QC codes that Ive used in past (eg. MOLCAS) dump a molden file during each step of optimization or MD. A single file would contain geometry and orbitals corresponding to a given step.

**Comment by agoetz** [comment]
I placed a PR to your molden branch. It fixes a few things
- correct geometry in [Atoms] block (geometry from last optimization step)
- only MOs from last geometry step in [MO] block (Molden displays MOs only for the geometry in the Atoms block)
- correct format in [GEOMETRIES] block
This now allows visualization of the optimization trajectory alongside with the energy plot from the [GEOCONV] block.
The only thing that does not work is display of data from the [SCFCONV] block. I tried to fix the format, but it still does not work. Not an issue imo.

@madu please check the PR in your repo and merge it if it looks good to you. I can then approve this PR.

**Comment by Madu86** [comment]
@agoetz I tested and merged your PR. Feel free to merge this PR when tests are done.

**Review by agoetz — APPROVED** [review]
Looking all good now.

**Pull request merged**